src/Functors.jl

```diff
@@ -143,6 +145,13 @@ julia> fmap(println, (i = twice, ii = 34, iii = [5, 6], iv = (twice, 34), v = 34
 34
 34.0
 (i = nothing, ii = nothing, iii = nothing, iv = (nothing, nothing), v = nothing)
+
+julia> d1 = Dict("x" => [1,2], "y" => 3);
+
+julia> d2 = Dict("x" => [4,5], "y" => 6);
+
+julia> fmap(+, d1, d2) == Dict("x" => [5, 7], "y" => 9)
```

Comment:
Since `_map(f, x::Dict, ys...)` mandates all `ys` have a superset of the keys `x` does and ignores any extra ones, modifying this example to reflect that would be good. Alternatively, we could add a `!!!` warning box in the docs about this behaviour with a separate example.

Reply:
I modified the example to show the behavior. Also added an extra test.